.circleci/config.yml

```diff
@@ -0,0 +1,176 @@
+
+  - &yarn_build
+    name: Building
+    command: yarn build
+
+  - &unit_test
+    name: Unit testing
+    # Limiting the workers of Jest to 10
+    # as the build otherwise dies due to resouce restrictions.
+    command: yarn test:ci --maxWorkers=10
+
+  - &unit_test_with_coverage
+    name: Unit testing (with coverage report)
+    command: yarn test:coverage:ci
+
+version: 2
+jobs:
+  greenkeeper_lockfile:
+    <<: *node8Environment
+    steps:
+      - checkout
+      - restore-cache: *restore_yarn_cache
+      - run:
+          name: Installing greenkeeper-lockfile-update
+          command: yarn global add greenkeeper-lockfile@1 --prefix ~/.yarn
+      - run: *yarn_install
+      - save_cache: *save_yarn_cache
+      - run:
+          name: Updating greenkeeper-lockfile
+          command: ~/.yarn/bin/greenkeeper-lockfile-update
+      - run:
+          name: Uploading greenkeeper-lockfile
+          command: ~/.yarn/bin/greenkeeper-lockfile-upload
+  lint:
+    <<: *node8Environment
+    steps:
+      - checkout
+      - restore-cache: *restore_yarn_cache
+      - run: *yarn_install
+      - run: *yarn_bootstrap
+      - run: *yarn_build
+      - save_cache: *save_yarn_cache
+      - run:
+          name: Linting
+          command: yarn lint
+  test_unit_node_4:
+    <<: *node4Environment
+    steps:
+      - checkout
+      - restore-cache: *restore_yarn_cache
+      - run: *yarn_install
+      - run: *yarn_bootstrap
+      - run: *yarn_build
+      - save_cache: *save_yarn_cache
+      - run: *unit_test
+  test_unit_node_6:
+    <<: *node6Environment
+    steps:
+      - checkout
+      - restore-cache: *restore_yarn_cache
+      - run: *yarn_install
+      - run: *yarn_bootstrap
+      - run: *yarn_build
+      - save_cache: *save_yarn_cache
+      - run: *unit_test
+  test_unit_node_8:
+    <<: *node8Environment
+    steps:
+      - checkout
+      - restore-cache: *restore_yarn_cache
+      - run: *yarn_install
+      - run: *yarn_bootstrap
+      - run: *yarn_build
+      - save_cache: *save_yarn_cache
+      - run: *unit_test_with_coverage
+  test_integration:
+    <<: *node8Environment
+    steps:
+      - checkout
+      - restore-cache: *restore_yarn_cache
+      - run: *yarn_install
+      - run: *yarn_bootstrap
+      - run: *yarn_build
+      - save_cache: *save_yarn_cache
+      - run:
+          name: Integration testing
+          command: yarn test:integration
+  prepare_release:
+    <<: *node8Environment
+    steps:
+      - checkout
+      - run:
+          name: Preparing release
+          command: ./.circleci/prepare-release-packages.sh
+  perform_release:
+    <<: *node8Environment
+    steps:
+      - checkout
+      - restore-cache: *restore_yarn_cache
+      - run: *yarn_install
+      - run: *yarn_bootstrap
+      - run: *yarn_build
+      - save_cache: *save_yarn_cache
+      - run:
+          name: Performing release
+          command: ./.circleci/perform-release-packages.sh
+
+workflows:
+  version: 2
```

Comment:
The version was already specified on [L51](https://github.com/commercetools/nodejs/pull/346/files#diff-1d37e48f9ceff6d8030570cd36286a61R51). Does it need to be specified here again?

Reply:
Yeah, I saw that on other configs. You specify the version for the build config itself and the workflows.